docs/guide/built-in-rules-callback.md

```diff
@@ -86,10 +86,10 @@ new Callback(
 > **Note:** Processing untrusted user input with `yaml_parse()` can be dangerous with certain settings. Please refer to
 > [yaml_parse docs] for more details. 
 
-### Making use of validation context for validating multiple attributes depending on each other
+### Using a validation context to validate interdependent attributes
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other

Reply:
- Making use of = Using 
- depending on each other = interdependent

Reply:
Result is shorter and a bit more readable. Meaning stays the same.